packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable react/forbid-dom-props */
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import * as KeyCode from 'keycode-js';
-import '../_elementPolyfill';
 import { injectIntl } from 'react-intl';
+import * as KeyCode from 'keycode-js';
 import classNames from 'classnames/bind';
+
 import ThemeContext from 'terra-theme-context';
+
 import WorklistDataGridUtils from '../utils/WorklistDataGridUtils';
 import styles from './Cell.module.scss';
+import ColumnContext from '../utils/ColumnContext';
+import '../_elementPolyfill';
```

Comment:
I don't think you need to add this polyfill.  You are not using the closest method.

Reply:
The polyfill was already being imported; we just reordered the imports.